keyboards/jotanck/config.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Jotix
+#define PRODUCT         Jotanck
+#define DESCRIPTION     A Jotantastic compact ortholinear keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 12
+
+/* pro_micro pin-out */
+#define MATRIX_ROW_PINS { B4, B5, B6, B2 }
+#define MATRIX_COL_PINS { F4, F5, F6, F7, B1, B3, D3, D2, D1, D0, D4, C6 }
+#define UNUSED_PINS
+
+/* leds */
+#define MATRIX_ROW_PINS { B4, B5, B6, B2 }
+#define QMK_LED E6
+#define BACKLIGHT_LEVELS 3
+
+/* tapping */
+#define TAPPING_TERM 250
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
```

Comment:
This doesn't need to be defined if you're using the default setting of Left Shift and Right Shift.

Lines 43-46 should be deleted.

Reply:
This.

Unless you're using something other than the default, this can and should be removed. 

And if you are using something else, you want to use `get_mods() & ` rather than `keyboard_report->mods == `